packages/helpermodules/subdata.py

```diff
@@ -635,10 +635,12 @@ def process_yourcharge_topic(self, var: yourcharge.YourCharge, msg: mqtt.MQTTMes
                     self.set_json_payload_class(var.data.yc_config, msg)
                     if var.data.yc_config.active:
                         self.event_control_algorithm_set.set()
-                        log.info("Setting YC ACTIVE mode = " + str(var.data.yc_config.active) + ": Switching to YourCharge control algorithm")
+                        log.info("Setting YC ACTIVE mode = " + str(var.data.yc_config.active)
+                                 + ": Switching to YourCharge control algorithm")
```

Comment:
Konsequent jeden Teilstring in eine eigene Zeile?

Reply:
Ehrlich gesagt hätte ich am liebsten mehr erlaubte Zeichen pro Zeile. 120 ist schon sehr konservativ für das was man heute so als Software-Entwickler an Bildschirm hat :)

Aber ich kann es natürlich gerne in drei Zeilen aufteilen bzw. würde ich in dem Fall eigentlich ein
```python
log.info(f"Setting YC ACTIVE mode = {var.data.yc_config.active}"
         + ": Switching to YourCharge control algorithm")
```
bevorzugen.

Reply:
> was man heute so als Software-Entwickler an Bildschirm hat

Nun, das ist ja auch ein open source-Projekt und ich beispielsweise beteilige mich am Laptop - mit dem vi... ;o)  
Letztendlich zählt aber, was das OWB-Team festgelegt hat, denn wenn man 3 Entwickler fragt, bekommt man 4 Meinungen dazu--- :D

> ich kann es natürlich gerne in drei Zeilen aufteilen

Wie gesagt finde ich Konsequenz  wichtig und an vielen anderen Stellen hast Du das ja auch so umgesetzt. 
Meine Meinung ist halt:  wenn man schon refactored, dann halt auch "richtig".   
Ich würde mich freuen, wenn Du meiner Einschätzung folgst.

Reply:
Ups, es ging mir gar nicht um eine Diskussion über die Coding-Richtlinien.

In Python bin ich totaler Anfänger und kenne die Pyhton-typischen coding-guidelines noch nicht.

Daher ging es mir eigentlich um die Klärung ob das mit `f(...)` auch eine Lösung wäre. Denn insbesondere in `subdata.py` ist vielfach `+str(...)+` verwendet.

Da das auch beantwortet wurde habe ich es entsprechend gepusht.